unify-front-end/supabase/migrations/20260518000001_interview_picker_cron.sql

```
@@ -0,0 +1,26 @@
+-- Enable extensions if not already
+CREATE EXTENSION IF NOT EXISTS pg_cron;
+CREATE EXTENSION IF NOT EXISTS pg_net;
+
+-- Remove any prior version of this job (safe re-apply)
+SELECT cron.unschedule('pick-interview-candidates-weekly')
+WHERE EXISTS (
+  SELECT 1 FROM cron.job WHERE jobname = 'pick-interview-candidates-weekly'
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)